airbyte-commons-worker/build.gradle

```diff
@@ -10,6 +10,7 @@ dependencies {
     implementation libs.bundles.micronaut
 
     implementation 'io.fabric8:kubernetes-client:5.12.2'
+    implementation 'com.auth0:java-jwt:3.19.2'
```

Comment:
curious, what is this for?

Reply:
This was necessary in order to move the ApiClientBeanFactory into commons-worker (to reuse it in both airbyte-workers and container-orchestrator). It does some token generation that needs this dependency.

Reply:
@gosusnp Could you expand a bit on what the issue might be? My change moved the `AirbyteAPiClientBeanFactory` to `airbyte-commons-worker`. I don't see `airbyte-workers` nor `airbyte-commons-worker` being a dependency of `airbyte-cron`.

Reply:
I was looking for an issue to link for better context but I cannot find it...
I think the problem we had is that micronaut discovers all singleton upon starts, so by having some services in a shared lib led us to start more services than intended. For example, the cron would pick up some beans from the workers and would effectively behave as an extra worker.

@jdpgrailsdev or @benmoriceau might have better context since they were more closely working on this.